jitify2.hpp

```diff
@@ -3673,11 +3616,7 @@ typedef unsigned short uint_least16_t;
 typedef unsigned int uint_least32_t;
 typedef unsigned long long uint_least64_t;
 typedef unsigned long long uintmax_t;
-#if defined _WIN32 || defined _WIN64
-typedef unsigned long long uintptr_t;  // optional
-#else  // not Windows
-typedef unsigned long uintptr_t;  // optional
-#endif
+
```

Comment:
I'd remove the blank lines for consistency with the previous densely-packed code.
```suggestion
```